src/components/video-editor/audio/waveform/waveform.worker.ts

```
@@ -0,0 +1,39 @@
+self.onmessage = (e: MessageEvent) => {
+	const { requestId, channels, samples } = e.data as {
+		requestId: number;
+		channels: Float32Array[];
+		samples: number;
+	};
+
+	if (!channels || channels.length === 0 || samples <= 0) {
+		const empty = new Float32Array(0);
+		(self as any).postMessage({ requestId, peaks: empty }, [empty.buffer]);
```

Biome (2.4.14)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)